src/explore-education-statistics-admin/src/components/form/FormEditor.tsx

```diff
@@ -43,9 +45,10 @@ export interface FormEditorProps {
   hideLabel?: boolean;
   hint?: string;
   id: string;
+  includePlugins?: ReadonlySet<PluginName>;
```

Comment:
We probably want to make this a little more flexible so it's less awkward to provide configs for very bespoke cases:

```ts
includePlugins?: ReadonlySet<PluginName> | Set<PluginName>;
```